inc/Perl/Critic/BuildUtilities.pm

```diff
@@ -40,7 +40,7 @@ sub required_module_versions {
         'Getopt::Long'                  => 0,
         'IO::String'                    => 0,
         'IPC::Open2'                    => 1,
-        'List::MoreUtils'               => 0.19,
+        'List::SomeUtils'               => 0.19,
```

Comment:
Is this really correct, 0.19 as well?  The [changelog](https://metacpan.org/dist/List-SomeUtils/changes) indicates that minimum Perl was only lowered to v5.6 in 0.400_010.